proxy/src/http/identity.rs

```diff
@@ -126,6 +153,16 @@ mod handler {
         let id = identity::get(&ctx.peer_api, &id)?;
         Ok(reply::json(&id))
     }
+
+    /// Retrieve the list of tracked identities.
+    pub async fn list_tracked_identities<R>(ctx: http::Ctx<R>) -> Result<impl Reply, Rejection>
+    where
+        R: Send + Sync,
+    {
+        let ctx = ctx.read().await;
+        let tracked_users = identity::list_tracked_identities(&ctx.peer_api)?;
```

Comment:
It seems like this endpoint only operates on tracked users, while we can and will also support tracking projects.

Shouldn't we reflect that in the endpoint by having, for now, `GET /tracked/users` and later `GET /tracked/projects`?

@FintanH?

Reply:
There's already a `/projects` endpoint. The next addition will be that we'll want something like `/projects/<user>`.

But I suppose this begs the question of "How should these endpoints be arranged?" I don't have any strong opinions or knowledge of organisation here :) cc @xla

Reply:
I think what makes this confusing is that for the first version, we want:
1) `/projects` - all projects created by the session user
2) `tbd` - projects tracked by the session user
3) `tbd` - projects the passed identity is involved in from our local view

3 is just temporary. currently it is neither projects-the-identity-is-tracking nor all-projects-the-identity-created

Reply:
Ah ya of course, so:
1. Is what we have today
2. Is a project that you track but haven't created a working copy, i.e. `heads` is empty. (I always forget about this :facepalm:)
3. Is what we're trying to build in this new feature, right?

Reply:
@FintanH yes to all three

Reply:
I summarized the paths in the [issue](https://github.com/radicle-dev/radicle-upstream/issues/741#issuecomment-672795977)